main.py

```diff
@@ -43,7 +44,20 @@ def api_label(my_path=None):
         if UPLOAD_LIMIT is not None and CURRENT_UPLOAD_SIZE >= UPLOAD_LIMIT:
             return Response(status=400, response='Reached upload limit')
         with open(os.path.join('static/data/', (my_path + '.json')), "w+") as outfile:
-            json.dump(request.get_json(), outfile, sort_keys=True, indent=4)
+            data = request.get_json()
+            json.dump(data, outfile, sort_keys=True, indent=4)
+            # Find the template for this folder
+            template = open(os.path.join('static/data/',(my_path[:my_path.rindex('/')] + '/template.json')))
+            # Convert the template json to something validate can understand.
+            template = json.load(template)
+            # Call the validate function from jsonschema to verify the file we were suppsoed to create conforms with the template for the file.
+            validate(data, template)
```

Comment:
Are you sure this works? The template and label are in completely different formats.... there's no way the jsonschema just figures this out...